output-core/src/models/Session.ts

```diff
@@ -1,6 +1,9 @@
-import { IsBoolean, IsObject } from '@jovotech/output';
+import { IsBoolean, IsObject, IsString } from '@jovotech/output';
 
 export class Session {
+  @IsString()
+  id?: string;
```

Comment:
If the property is optional a `IsOptional` decorator is required. Otherwise, this will always throw an error if no `id` was passed. Please add `@IsOptional()`. Maybe is also makes sense to check if the `id` is not empty, but this may not be mandatory.